lib/buy/buy_quote.dart

```diff
@@ -0,0 +1,226 @@
+import 'package:cake_wallet/buy/buy_provider.dart';
+import 'package:cake_wallet/core/selectable_option.dart';
+import 'package:cake_wallet/entities/provider_types.dart';
+
+enum ProviderRecommendation { bestRate, lowKyc, successRate }
+
+extension RecommendationTitle on ProviderRecommendation {
+  String get title {
+    switch (this) {
+      case ProviderRecommendation.bestRate:
+        return 'BEST RATE';
+      case ProviderRecommendation.lowKyc:
+        return 'LOW KYC';
+      case ProviderRecommendation.successRate:
+        return 'SUCCESS RATE';
+    }
+  }
+}
+
+ProviderRecommendation? getRecommendationFromString(String title) {
+  switch (title) {
+    case 'BEST RATE':
```

Comment:
yes, this is what was in the Figma mockup

Reply:
Figma? this is used for fetching the recommendations returned by onramper?!

Reply:
Ah, you were asking about the expected response value. From Onramper, we are getting ‘BestPrice’, but we ignore it anyway because we set the best rate ourselves here:
`recommendations.insert(0, ProviderRecommendation.bestRate);`
in the view model.